mod/invitefriends/actions/invitefriends/invite.php

```diff
@@ -13,6 +13,9 @@
 }
 
 $site = elgg_get_site_entity();
+// create the from address
+$site_email = new \Zend\Mail\Address($site->getEmailAddress(), $site->getDisplayName());
```

Comment:
I think we have an Elgg helper class. We should probably avoid using vendor classes in public API